src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -436,16 +441,15 @@ export default class CopyNumberTableWrapper extends React.Component<
             );
         },
     });
+}
 
-    // @action.bound
-    // private handleOncoKbIconModeToggle(mergeIcons: boolean) {
-    //     this.mergeOncoKbIcons = mergeIcons;
-    //     updateOncoKbIconStyle({ mergeIcons });
-    //
-    //     // we need to set the OncoKB width on the next render cycle, otherwise it is not updated yet
-    //     calculateOncoKbContentWidthOnNextFrame(
-    //         ANNOTATION_ELEMENT_ID,
-    //         width => (this.oncokbWidth = width || DEFAULT_ONCOKB_CONTENT_WIDTH)
-    //     );
-    // }
+function createCnaNamespaceColumns(
+    config?: NamespaceColumnConfig
+): CNATableColumn[] {
+    const namespaceColumnRecords = createNamespaceColumns(config);
+    const namespaceColumns = Object.values(
```

Comment:
Lodash will allow you to iterate over the members of an object.
```
_.forEach(namespaceColumnRecords,c=>v.bisible = false)
```

Reply:
Why use lodash
```
_.forEach(namespaceColumns,c => c.visible = false)
```
when the plain js version is just as succinct?
```
namespaceColumns.forEach(c => c.visible = false)
```

Reply:
Changed in [4926763](https://github.com/cBioPortal/cbioportal-frontend/commit/49267637fe0a3a2d8241df58de3dd3dd8321a7f9)

Reply:
lodash allow you to iterate directly over the members (key,value) of an object so you don't have to first use Object.values just to get an array.    So you can do 
```_.forEach(namespaceColumnRecords)```
i think you already use _.forIn elsewhere, so, same idea.

if you are. already dealing with an array, then sure, no need to use lodash.

Reply:
i see here you are transforming it into an array for return.  in that case _.map will also work on an object and returns an array.